curator-client/src/main/java/org/apache/curator/CuratorZookeeperClient.java

```diff
@@ -213,18 +213,29 @@ public void start() throws Exception
 
         state.start();
     }
-
+    
     /**
      * Close the client
      */
-    public void close()
+    public void close() {
+        close(0);
+    }
+            
+    /**
+     * Close this client object as the {@link #close() } method.
+     * This method will wait for internal resources to be released.
+     * 
+     * @param waitForShutdownTimeoutMs timeout (in milliseconds) to wait for resources to be released.
+     *                  Use zero or a negative value to skip the wait.
+     */
+    public void close(int waitForShutdownTimeoutMs)
```

Comment:
@Randgalt in internal ZK handle we are returning a boolean I think that in Curator this is not necessary.
I am leaving *void*